src/app/lib/globalStyles/index.jsx

```diff
@@ -22,9 +22,11 @@ const GlobalStyleComponent = createGlobalStyle`
     ${({ fonts }) => fonts && createFontStyles(fonts)}
   `;
 
-const GlobalStyle = () => {
-  const { fonts } = useContext(ServiceContext);
-  return <GlobalStyleComponent fonts={fonts} />;
+// eslint-disable-next-line react/prop-types
+const GlobalStyle = ({ fonts }) => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const styleFonts = fonts || useContext(ServiceContext).fonts;
+  return <GlobalStyleComponent fonts={styleFonts} />;
```

Comment:
I'm alarmed that we are having to suppress two linter rules just to support this change, esp `rules-of-hooks`. What are the alternatives? Can we refactor to not violate these rules?